src/Microsoft.IdentityModel.Tokens/LogMessages.cs

```diff
@@ -197,6 +197,8 @@ internal static class LogMessages
         public const string IDX10693 = "IDX10693: RSACryptoServiceProvider doesn't support the RSASSA-PSS signature algorithm. The list of supported algorithms is available here: https://aka.ms/IdentityModel/supported-algorithms";
         public const string IDX10694 = "IDX10694: JsonWebKeyConverter threw attempting to convert JsonWebKey: '{0}'. Exception: '{1}'.";
         public const string IDX10695 = "IDX10695: Unable to create a JsonWebKey from an ECDsa object. Required ECParameters structure is not supported by .NET Framework < 4.7.";
+        public const string IDX10696 = "IDX10696: The algorithm '{0}' is not in the user defined accepted list of algorithms.";
+        public const string IDX10697 = "IDX10697: The user defined 'Delegate' AcceptedAlgorithmValidator specified on TokenValidationParameters returned false when validating the algorithm for Algorithm: '{0}', SecurityKey: '{1}'.";
```

Comment:
nit: Do we need to capitalize 'Delegate' and 'Algorithm' here?

Also, can the sentence be simplified by removing 'for Algorithm' at the end?

Reply:
This is a fair question. I was trying to keep this consistent with IDX10505, IDX10506, IDX10508, and IDX10509. Will take another look